src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RedirectTest.java

```diff
@@ -0,0 +1,27 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.JavaScriptRedirectPage;
```

Comment:
Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.RedirectPage;`